courses/additional_modules/labs/answers/advanced_privacy.txt

```diff
@@ -0,0 +1,157 @@
+
```

Comment:
I assume you're planning to include a CI script that verifies that these source-code examples compile fine and generate the correct output.

Reply:
In any case, I've just verified that this source-code example is ok.

Reply:
Probably at some time in the (near) future.
Leo is already working on a CI script to produce / verify the documents, so I'll probably ask him to add this as well